io/js/src/main/scala/fs2/io/net/tls/TLSSocketPlatform.scala

```diff
@@ -84,6 +84,7 @@ private[tls] trait TLSSocketCompanionPlatform { self: TLSSocket.type =>
         readable
       ).race(errorDef.get.flatMap(F.raiseError[SuspendedStream[F, Byte]]).toResource)
         .map(_.merge)
+      _ <- errorDef.tryGet.map(_.toLeft(())).rethrow.toResource
```

Comment:
@ChristopherDavenport the error is raised here (it is captured in an `'error'` event listener above).

Reply:
Ok, this makes sense and looks good to me then.